design/dsl/endpoint_test.go

```diff
@@ -57,7 +55,7 @@ func TestEndpoint(t *testing.T) {
 				if len(endpoint.Errors) != 2 {
 					t.Errorf("expected %d error definitions but got %d ", 2, len(endpoint.Errors))
```

Comment:
This should be a `Fatalf` otherwise the test will panic when retrieving `endpoints[1]` below.